__tests__/notifications.ts

```diff
@@ -551,6 +552,9 @@ describe('mutation muteNotificationPreference', () => {
       .findOneBy(params);
 
     expect(muted).toBeTruthy();
+    expect(muted.type).toEqual(
+      notificationPreferenceMap[params.notificationType],
+    );
```

Comment:
This test I think covers the type check during insert 👍

Reply:
Yes, this is the one I also used to make the test fail, and applied the fix to make this go green.

Reply:
cc @idoshamun

Reply:
Missed that!